### ISSUE TITLE
Wording is a bit hard to follow in one phrase

### ISSUE THREAD

**Issue by jfbenson** [dotnet/docs#5953]
"members accessed through one of the interfaces the current collections implement"

I am having trough understanding what "the current collections" is referring to? Is it referring to System.Collections? If so could you replace "the current collections" with "System.Collections"? I think this will also prevent ambiguity in the future because the word 'current' is subject to change as time moves on.

Thanks,
Jack

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0e353074-f9d0-51fe-ec60-29993cd47909
* Version Independent ID: 339af9d9-70ea-fe53-041d-56a2cfccff4c
* Content: [System.Collections.Concurrent Namespace](https://docs.microsoft.com/en-us/dotnet/api/system.collections.concurrent?view=netframework-4.7.2)
* Content Source: [xml/ns-System.Collections.Concurrent.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/ns-System.Collections.Concurrent.xml)
* Product: **.net**
* GitHub Login: @mairaw
* Microsoft Alias: **mairaw**

**Comment by rpetrusha** [comment]
I agree, @jfbenson, that it's a bit hard to follow. I think that what was intended is, "However, access to elements of a collection object through extension methods or through explicit interface implementations are not guaranteed to be thread-safe and may need to be synchronized by the caller." Does that help?
We'll clarify this in the documentation this month.

**Comment by jfbenson** [comment]
Sounds good to me. Thanks Ron!
